tests/system-tests/o-cloud/internal/ocloudcommon/sno-provisioning-ibi.go

```diff
@@ -0,0 +1,408 @@
+//
+//nolint:funlen
+func generateBaseImage(ctx SpecContext) {
+	By("Generating a base image from seed SNO")
+
+	provisioningRequest := VerifyProvisionSnoCluster(
+		OCloudConfig.TemplateName,
+		OCloudConfig.TemplateVersionSeed,
+		OCloudConfig.NodeClusterName1,
+		OCloudConfig.OCloudSiteID,
+		ocloudparams.PolicyTemplateParameters,
+		ocloudparams.ClusterInstanceParameters1)
+
+	node, nodePool, namespace, clusterInstance := VerifyAndRetrieveAssociatedCRsForAI(
+		provisioningRequest.Object.Name, OCloudConfig.ClusterName1, ctx)
+
+	VerifyAllPoliciesInNamespaceAreCompliant(namespace.Object.Name, ctx, nil, nil)
+	glog.V(ocloudparams.OCloudLogLevel).Infof("All the policies are compliant")
+
+	VerifyProvisioningRequestIsFulfilled(provisioningRequest)
+	glog.V(ocloudparams.OCloudLogLevel).Infof("Provisioning request %s is fulfilled", provisioningRequest.Object.Name)
+
+	snoAPIClient := CreateSnoAPIClient(OCloudConfig.ClusterName1)
+
+	By("Verifying the SR-IOV network node states")
+
+	networkNodeStates, err := sriov.ListNetworkNodeState(snoAPIClient, ocloudparams.SriovNamespace)
+	Expect(err).NotTo(HaveOccurred(),
+		fmt.Sprintf("Error getting the list of SR-IOV network node states in namespace %s", ocloudparams.SriovNamespace))
+
+	for _, networkNodeState := range networkNodeStates {
+		err = networkNodeState.WaitUntilSyncStatus("Succeeded", 30*time.Minute)
+		Expect(err).NotTo(HaveOccurred(),
+			fmt.Sprintf("SR-IOV network node state %s did not succeeded", networkNodeState.Objects.Name))
+	}
+
+	By("Detaching the seed SNO from the hub")
+
+	cluster, err := ocm.PullManagedCluster(HubAPIClient, OCloudConfig.ClusterName1)
+
+	if err == nil {
+		err = cluster.DeleteAndWait(time.Minute * 10)
+		Expect(err).NotTo(HaveOccurred(),
+			fmt.Sprintf("Error deleting managed cluster %s", OCloudConfig.ClusterName1))
+	}
+
+	By("Creating a seedgen secret in the LCA namespace")
+
+	secret := secret.NewBuilder(snoAPIClient, "seedgen", ocloudparams.LifecycleAgentNamespace, corev1.SecretTypeOpaque)
+	data := make(map[string][]byte)
+	data["seedAuth"] = []byte(OCloudConfig.LocalRegistryAuth)
+	secret.WithData(data)
+	_, err = secret.Create()
+	Expect(err).NotTo(HaveOccurred(),
+		fmt.Sprintf("Error creating seedgen secret in namespace %s: %v", ocloudparams.LifecycleAgentNamespace, err))
+
+	By("Creating a seed generator")
+
+	seedGenerator := lca.NewSeedGeneratorBuilder(snoAPIClient, ocloudparams.SeedGeneratorName)
+	seedGenerator.WithSeedImage(OCloudConfig.SeedImage)
+	seedGenerator, err = seedGenerator.Create()
+	Expect(err).NotTo(HaveOccurred(),
+		fmt.Sprintf("Error creating seedgenerator seedimage: %v", err))
+
+	_, err = seedGenerator.WaitUntilComplete(30 * time.Minute)
+	Expect(err).NotTo(HaveOccurred(),
+		fmt.Sprintf("Seedgenerator seedimage did not completed: %v", err))
+
+	By("Creating the base image")
+
+	_, err = shell.ExecuteCmd(ocloudparams.CreateImageBasedInstallationConfig)
```

Comment:
This becomes problematic when running the tests using the existing eco-gotests container image since it's missing the openshift-install binary. We can probably add it to the ran-du container image: https://github.com/openshift-kni/eco-gotests/blob/main/images/system-tests/ran-du/Dockerfile

Reply:
@mcornea I have created a new script in ran-integration repo to run the ORAN tests directly from the hub
https://gitlab.cee.redhat.com/ran/ran-integration/-/merge_requests/484

Reply:
I have added to the ran-du container image the installation of openshift-install and buildah